<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>alevinQC • alevinQC</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.3.7/cosmo/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha256-U5ZEeKfGNOja007MMD3YBI0A3OSZOQbeG6z2f2Y0hu8=" crossorigin="anonymous"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.7.1/css/all.min.css" integrity="sha256-nAmazAk6vS34Xqo0BSrTb+abbtFlgsFK7NKSi6o7Y78=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.7.1/css/v4-shims.min.css" integrity="sha256-6qHlizsOWFskGlwVOKuns+D1nB6ssZrHQrNj1wGplHc=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.4/clipboard.min.js" integrity="sha256-FiZwavyI2V6+EXO1U+xzLG3IKldpiTFf3153ea9zikQ=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4/headroom.min.js" integrity="sha256-DJFC1kqIhelURkuza0AvYal5RxMtpzLjFhsnVIeuk+U=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.9.4/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="alevinQC">
<meta property="og:description" content="">
<meta name="twitter:card" content="summary">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body>
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">alevinQC</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.3.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../index.html">
    <span class="fas fa fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/alevinqc.html">alevinQC</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/csoneson/alevinQC">
    <span class="fab fa fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1>alevinQC</h1>
                        <h4 class="author">Charlotte Soneson</h4>
            
            <h4 class="date">2020-03-17</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/csoneson/alevinQC/blob/master/vignettes/alevinqc.Rmd"><code>vignettes/alevinqc.Rmd</code></a></small>
      <div class="hidden name"><code>alevinqc.Rmd</code></div>

    </div>

    
        <div class="abstract">
      <p class="abstract">Abstract</p>
      <em><a href="https://bioconductor.org/packages/3.11/alevinQC">alevinQC</a></em> reads output files from alevin and generates summary reports.
    </div>
    
<div id="introduction" class="section level1">
<h1 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h1>
<p>The purpose of the <em><a href="https://bioconductor.org/packages/3.11/alevinQC">alevinQC</a></em> package is to generate a summary QC report based on the output of an <a href="https://salmon.readthedocs.io/en/latest/alevin.html">alevin</a> <span class="citation">(Srivastava et al. 2018)</span> run. The QC report can be generated as a html or pdf file, or launched as a shiny application.</p>
</div>
<div id="installation" class="section level1">
<h1 class="hasAnchor">
<a href="#installation" class="anchor"></a>Installation</h1>
<p><code>alevinQC</code> can be installed using the <code>BiocManager</code> CRAN package.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="cf">if</span> (<span class="op">!</span><span class="kw"><a href="https://rdrr.io/r/base/ns-load.html">requireNamespace</a></span>(<span class="st">"BiocManager"</span>, <span class="dt">quietly =</span> <span class="ot">TRUE</span>))</a>
<a class="sourceLine" id="cb1-2" data-line-number="2">    <span class="kw"><a href="https://rdrr.io/r/utils/install.packages.html">install.packages</a></span>(<span class="st">"BiocManager"</span>)</a>
<a class="sourceLine" id="cb1-3" data-line-number="3">BiocManager<span class="op">::</span><span class="kw"><a href="https://rdrr.io/pkg/BiocManager/man/install.html">install</a></span>(<span class="st">"alevinQC"</span>)</a></code></pre></div>
<p>After installation, load the package into the R session.</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span>(alevinQC)</a></code></pre></div>
<p>Note that in order to process output from Salmon v0.14 or later, you need Alevin v1.1 or later.</p>
</div>
<div id="assumed-output-directory-structure" class="section level1">
<h1 class="hasAnchor">
<a href="#assumed-output-directory-structure" class="anchor"></a>Assumed output directory structure</h1>
<p>For more information about running alevin, we refer to the <a href="https://salmon.readthedocs.io/en/latest/alevin.html">documentation</a>. When invoked, alevin generates several output files in the specified output directory. <em><a href="https://bioconductor.org/packages/3.11/alevinQC">alevinQC</a></em> assumes that this structure is retained, and will return an error if it isn’t - thus, it is not recommended to move or rename the output files from alevin. <em><a href="https://bioconductor.org/packages/3.11/alevinQC">alevinQC</a></em> assumes that the following files (in the indicated structure) are available in the provided <code>baseDir</code> (note that currently, in order to generate the full set of files, alevin must be invoked with the <code>--dumpFeatures</code> flag).</p>
<p>For alevin versions before 0.14:</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb3-1" data-line-number="1">baseDir</a>
<a class="sourceLine" id="cb3-2" data-line-number="2">  <span class="op">|--</span>alevin</a>
<a class="sourceLine" id="cb3-3" data-line-number="3">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>featureDump.txt</a>
<a class="sourceLine" id="cb3-4" data-line-number="4">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>filtered_cb_frequency.txt</a>
<a class="sourceLine" id="cb3-5" data-line-number="5">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>MappedUmi.txt</a>
<a class="sourceLine" id="cb3-6" data-line-number="6">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>quants_mat_cols.txt</a>
<a class="sourceLine" id="cb3-7" data-line-number="7">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>quants_mat_rows.txt</a>
<a class="sourceLine" id="cb3-8" data-line-number="8">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>quants_mat.gz</a>
<a class="sourceLine" id="cb3-9" data-line-number="9">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>raw_cb_frequency.txt</a>
<a class="sourceLine" id="cb3-10" data-line-number="10">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>whitelist.txt</a>
<a class="sourceLine" id="cb3-11" data-line-number="11">  <span class="op">|--</span>aux_info</a>
<a class="sourceLine" id="cb3-12" data-line-number="12">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>meta_info.json</a>
<a class="sourceLine" id="cb3-13" data-line-number="13">  <span class="op">|--</span>cmd_info.json</a></code></pre></div>
<p>For alevin version 0.14 and later:</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" data-line-number="1">baseDir</a>
<a class="sourceLine" id="cb4-2" data-line-number="2">  <span class="op">|--</span>alevin</a>
<a class="sourceLine" id="cb4-3" data-line-number="3">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>featureDump.txt</a>
<a class="sourceLine" id="cb4-4" data-line-number="4">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>raw_cb_frequency.txt</a>
<a class="sourceLine" id="cb4-5" data-line-number="5">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>whitelist.txt</a>
<a class="sourceLine" id="cb4-6" data-line-number="6">  <span class="op">|--</span>aux_info</a>
<a class="sourceLine" id="cb4-7" data-line-number="7">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>meta_info.json</a>
<a class="sourceLine" id="cb4-8" data-line-number="8">  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>alevin_meta_info.json</a>
<a class="sourceLine" id="cb4-9" data-line-number="9">  <span class="op">|--</span>cmd_info.json</a></code></pre></div>
</div>
<div id="check-that-all-required-alevin-files-are-available" class="section level1">
<h1 class="hasAnchor">
<a href="#check-that-all-required-alevin-files-are-available" class="anchor"></a>Check that all required alevin files are available</h1>
<p>The report generation functions (see below) will check that all the required files are available in the provided base directory. However, you can also call the function <code><a href="../reference/checkAlevinInputFiles.html">checkAlevinInputFiles()</a></code> to run the check manually. If one or more files are missing, the function will raise an error indicating the missing file(s).</p>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb5-1" data-line-number="1">baseDir &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/system.file.html">system.file</a></span>(<span class="st">"extdata/alevin_example_v0.14"</span>, <span class="dt">package =</span> <span class="st">"alevinQC"</span>)</a>
<a class="sourceLine" id="cb5-2" data-line-number="2"><span class="kw"><a href="../reference/checkAlevinInputFiles.html">checkAlevinInputFiles</a></span>(<span class="dt">baseDir =</span> baseDir)</a>
<a class="sourceLine" id="cb5-3" data-line-number="3"><span class="co">#&gt; [1] "v0.14"</span></a></code></pre></div>
</div>
<div id="generate-qc-report" class="section level1">
<h1 class="hasAnchor">
<a href="#generate-qc-report" class="anchor"></a>Generate QC report</h1>
<p>The <code><a href="../reference/alevinQCReport.html">alevinQCReport()</a></code> function generates the QC report from the alevin output. Depending on the file extension of the <code>outputFile</code> argument, and the value of <code>outputFormat</code>, the function can generate either an html report or a pdf report.</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" data-line-number="1">outputDir &lt;-<span class="st"> </span><span class="kw"><a href="https://rdrr.io/r/base/tempfile.html">tempdir</a></span>()</a>
<a class="sourceLine" id="cb6-2" data-line-number="2"><span class="kw"><a href="../reference/alevinQCReport.html">alevinQCReport</a></span>(<span class="dt">baseDir =</span> baseDir, <span class="dt">sampleId =</span> <span class="st">"testSample"</span>, </a>
<a class="sourceLine" id="cb6-3" data-line-number="3">               <span class="dt">outputFile =</span> <span class="st">"alevinReport.html"</span>, </a>
<a class="sourceLine" id="cb6-4" data-line-number="4">               <span class="dt">outputFormat =</span> <span class="st">"html_document"</span>,</a>
<a class="sourceLine" id="cb6-5" data-line-number="5">               <span class="dt">outputDir =</span> outputDir, <span class="dt">forceOverwrite =</span> <span class="ot">TRUE</span>)</a></code></pre></div>
</div>
<div id="create-shiny-app" class="section level1">
<h1 class="hasAnchor">
<a href="#create-shiny-app" class="anchor"></a>Create shiny app</h1>
<p>In addition to static reports, <em><a href="https://bioconductor.org/packages/3.11/alevinQC">alevinQC</a></em> can also generate a shiny application, containing the same summary figures as the pdf and html reports.</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb7-1" data-line-number="1">app &lt;-<span class="st"> </span><span class="kw"><a href="../reference/alevinQCShiny.html">alevinQCShiny</a></span>(<span class="dt">baseDir =</span> baseDir, <span class="dt">sampleId =</span> <span class="st">"testSample"</span>)</a></code></pre></div>
<p>Once created, the app can be launched using the <code>runApp()</code> function from the <em><a href="https://CRAN.R-project.org/package=shiny">shiny</a></em> package.</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb8-1" data-line-number="1">shiny<span class="op">::</span><span class="kw"><a href="https://rdrr.io/pkg/shiny/man/runApp.html">runApp</a></span>(app)</a></code></pre></div>
</div>
<div id="generate-individual-plots" class="section level1">
<h1 class="hasAnchor">
<a href="#generate-individual-plots" class="anchor"></a>Generate individual plots</h1>
<p>The individual plots included in the QC reports can also be independently generated. To do so, we must first read the alevin output into an R object.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb9-1" data-line-number="1">alevin &lt;-<span class="st"> </span><span class="kw"><a href="../reference/readAlevinQC.html">readAlevinQC</a></span>(<span class="dt">baseDir =</span> baseDir)</a></code></pre></div>
<p>The resulting list contains three entries:</p>
<ul>
<li>
<code>cbTable</code>: a <code>data.frame</code> with various inferred characteristics of the individual cell barcodes</li>
<li>
<code>summaryTables</code>: a list of <code>data.frame</code>s with summary information about the full data set, the initial set of whitelisted cells and the final set of whitelisted cells, respectively</li>
<li>
<code>versionTable</code>: a <code>matrix</code> with information about the invokation of alevin</li>
</ul>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb10-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/utils/head.html">head</a></span>(alevin<span class="op">$</span>cbTable)</a>
<a class="sourceLine" id="cb10-2" data-line-number="2"><span class="co">#&gt;                 CB originalFreq ranking collapsedFreq nbrMappedUMI</span></a>
<a class="sourceLine" id="cb10-3" data-line-number="3"><span class="co">#&gt; 1 GACTGCGAGGGCATGT       121577       1        123419       104128</span></a>
<a class="sourceLine" id="cb10-4" data-line-number="4"><span class="co">#&gt; 2 GGTGCGTAGGCTACGA       110467       2        111987        93608</span></a>
<a class="sourceLine" id="cb10-5" data-line-number="5"><span class="co">#&gt; 3 ATGAGGGAGTAGTGCG       106446       3        108173        88481</span></a>
<a class="sourceLine" id="cb10-6" data-line-number="6"><span class="co">#&gt; 4 ACTGTCCTCATGCTCC       104794       4        106085        81879</span></a>
<a class="sourceLine" id="cb10-7" data-line-number="7"><span class="co">#&gt; 5 CGAACATTCTGATACG       104616       5        106072        84395</span></a>
<a class="sourceLine" id="cb10-8" data-line-number="8"><span class="co">#&gt; 6 ACTGTCCCATATGGTC        99208       6        100776        81066</span></a>
<a class="sourceLine" id="cb10-9" data-line-number="9"><span class="co">#&gt;   totalUMICount mappingRate dedupRate  MeanByMax nbrGenesAboveZero</span></a>
<a class="sourceLine" id="cb10-10" data-line-number="10"><span class="co">#&gt; 1         73312    0.843695  0.295943 0.00735194              7512</span></a>
<a class="sourceLine" id="cb10-11" data-line-number="11"><span class="co">#&gt; 2         66002    0.835883  0.294911 0.00783094              7522</span></a>
<a class="sourceLine" id="cb10-12" data-line-number="12"><span class="co">#&gt; 3         62196    0.817958  0.297069 0.00832595              7081</span></a>
<a class="sourceLine" id="cb10-13" data-line-number="13"><span class="co">#&gt; 4         57082    0.771824  0.302849 0.00619664              6956</span></a>
<a class="sourceLine" id="cb10-14" data-line-number="14"><span class="co">#&gt; 5         58547    0.795639  0.306274 0.00743685              7347</span></a>
<a class="sourceLine" id="cb10-15" data-line-number="15"><span class="co">#&gt; 6         56534    0.804418  0.302618 0.00947029              6841</span></a>
<a class="sourceLine" id="cb10-16" data-line-number="16"><span class="co">#&gt;   nbrGenesAboveMean ArborescenceCount inFinalWhiteList inFirstWhiteList</span></a>
<a class="sourceLine" id="cb10-17" data-line-number="17"><span class="co">#&gt; 1              1237           1.42034             TRUE             TRUE</span></a>
<a class="sourceLine" id="cb10-18" data-line-number="18"><span class="co">#&gt; 2              1238           1.41826             TRUE             TRUE</span></a>
<a class="sourceLine" id="cb10-19" data-line-number="19"><span class="co">#&gt; 3              1151           1.42262             TRUE             TRUE</span></a>
<a class="sourceLine" id="cb10-20" data-line-number="20"><span class="co">#&gt; 4               957           1.43441             TRUE             TRUE</span></a>
<a class="sourceLine" id="cb10-21" data-line-number="21"><span class="co">#&gt; 5              1238           1.44149             TRUE             TRUE</span></a>
<a class="sourceLine" id="cb10-22" data-line-number="22"><span class="co">#&gt; 6              1068           1.43393             TRUE             TRUE</span></a></code></pre></div>
<div class="sourceCode" id="cb11"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb11-1" data-line-number="1">knitr<span class="op">::</span><span class="kw"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span>(alevin<span class="op">$</span>summaryTables<span class="op">$</span>fullDataset)</a></code></pre></div>
<table class="table"><tbody>
<tr class="odd">
<td align="left">Total number of processed reads</td>
<td align="left">7197662</td>
</tr>
<tr class="even">
<td align="left">Number of reads with Ns</td>
<td align="left">35362</td>
</tr>
<tr class="odd">
<td align="left">Number of reads with valid cell barcode (no Ns)</td>
<td align="left">7162300</td>
</tr>
<tr class="even">
<td align="left">Number of noisy CB reads</td>
<td align="left">1003624</td>
</tr>
<tr class="odd">
<td align="left">Number of noisy UMI reads</td>
<td align="left">266</td>
</tr>
<tr class="even">
<td align="left">Total number of observed cell barcodes</td>
<td align="left">188613</td>
</tr>
</tbody></table>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb12-1" data-line-number="1">knitr<span class="op">::</span><span class="kw"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span>(alevin<span class="op">$</span>summaryTables<span class="op">$</span>initialWhitelist)</a></code></pre></div>
<table class="table"><tbody>
<tr class="odd">
<td align="left">Number of barcodes (initial whitelist)</td>
<td align="left">100</td>
</tr>
<tr class="even">
<td align="left">Fraction reads in barcodes (initial whitelist)</td>
<td align="left">84.64%</td>
</tr>
<tr class="odd">
<td align="left">Mean number of reads per cell (initial whitelist)</td>
<td align="left">60620</td>
</tr>
<tr class="even">
<td align="left">Median number of reads per cell (initial whitelist)</td>
<td align="left">58132</td>
</tr>
<tr class="odd">
<td align="left">Median number of detected genes per cell (initial whitelist)</td>
<td align="left">5268</td>
</tr>
<tr class="even">
<td align="left">Median UMI count per cell (initial whitelist)</td>
<td align="left">31353</td>
</tr>
</tbody></table>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb13-1" data-line-number="1">knitr<span class="op">::</span><span class="kw"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span>(alevin<span class="op">$</span>summaryTables<span class="op">$</span>finalWhitelist)</a></code></pre></div>
<table class="table"><tbody>
<tr class="odd">
<td align="left">Number of barcodes (final whitelist)</td>
<td align="left">95</td>
</tr>
<tr class="even">
<td align="left">Fraction reads in barcodes (final whitelist)</td>
<td align="left">82.39%</td>
</tr>
<tr class="odd">
<td align="left">Mean number of reads per cell (final whitelist)</td>
<td align="left">62118</td>
</tr>
<tr class="even">
<td align="left">Median number of reads per cell (final whitelist)</td>
<td align="left">58725</td>
</tr>
<tr class="odd">
<td align="left">Median number of detected genes per cell (final whitelist)</td>
<td align="left">5343</td>
</tr>
<tr class="even">
<td align="left">Median UMI count per cell (final whitelist)</td>
<td align="left">32028</td>
</tr>
</tbody></table>
<div class="sourceCode" id="cb14"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb14-1" data-line-number="1">knitr<span class="op">::</span><span class="kw"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span>(alevin<span class="op">$</span>versionTable)</a></code></pre></div>
<table class="table"><tbody>
<tr class="odd">
<td align="left">Start time</td>
<td align="left">Thu May 30 13:06:55 2019</td>
</tr>
<tr class="even">
<td align="left">Salmon version</td>
<td align="left">0.14.0</td>
</tr>
<tr class="odd">
<td align="left">Index</td>
<td align="left">/mnt/scratch5/avi/alevin/data/mohu/salmon_index</td>
</tr>
<tr class="even">
<td align="left">R1file</td>
<td align="left">/mnt/scratch5/avi/alevin/data/10x/v2/mohu/100/all_bcs.fq</td>
</tr>
<tr class="odd">
<td align="left">R2file</td>
<td align="left">/mnt/scratch5/avi/alevin/data/10x/v2/mohu/100/all_reads.fq</td>
</tr>
<tr class="even">
<td align="left">tgMap</td>
<td align="left">/mnt/scratch5/avi/alevin/data/mohu/gtf/txp2gene.tsv</td>
</tr>
</tbody></table>
<p>The plots can now be generated using the dedicated plotting functions provided with <em><a href="https://bioconductor.org/packages/3.11/alevinQC">alevinQC</a></em> (see the help file for the respective function for more information).</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb15-1" data-line-number="1"><span class="kw"><a href="../reference/plotAlevinKneeRaw.html">plotAlevinKneeRaw</a></span>(alevin<span class="op">$</span>cbTable)</a></code></pre></div>
<p><img src="alevinqc_files/figure-html/v13-1.png" width="700"></p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb16-1" data-line-number="1"><span class="kw"><a href="../reference/plotAlevinBarcodeCollapse.html">plotAlevinBarcodeCollapse</a></span>(alevin<span class="op">$</span>cbTable)</a></code></pre></div>
<p><img src="alevinqc_files/figure-html/v13-2.png" width="700"></p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb17-1" data-line-number="1"><span class="kw"><a href="../reference/plotAlevinQuant.html">plotAlevinQuant</a></span>(alevin<span class="op">$</span>cbTable)</a></code></pre></div>
<p><img src="alevinqc_files/figure-html/v13-3.png" width="700"></p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb18-1" data-line-number="1"><span class="kw"><a href="../reference/plotAlevinKneeNbrGenes.html">plotAlevinKneeNbrGenes</a></span>(alevin<span class="op">$</span>cbTable)</a></code></pre></div>
<p><img src="alevinqc_files/figure-html/v13-4.png" width="700"></p>
</div>
<div id="session-info" class="section level1">
<h1 class="hasAnchor">
<a href="#session-info" class="anchor"></a>Session info</h1>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb19-1" data-line-number="1"><span class="kw"><a href="https://rdrr.io/r/utils/sessionInfo.html">sessionInfo</a></span>()</a>
<a class="sourceLine" id="cb19-2" data-line-number="2"><span class="co">#&gt; R Under development (unstable) (2020-03-13 r77948)</span></a>
<a class="sourceLine" id="cb19-3" data-line-number="3"><span class="co">#&gt; Platform: x86_64-pc-linux-gnu (64-bit)</span></a>
<a class="sourceLine" id="cb19-4" data-line-number="4"><span class="co">#&gt; Running under: Ubuntu 16.04.6 LTS</span></a>
<a class="sourceLine" id="cb19-5" data-line-number="5"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb19-6" data-line-number="6"><span class="co">#&gt; Matrix products: default</span></a>
<a class="sourceLine" id="cb19-7" data-line-number="7"><span class="co">#&gt; BLAS:   /home/travis/R-bin/lib/R/lib/libRblas.so</span></a>
<a class="sourceLine" id="cb19-8" data-line-number="8"><span class="co">#&gt; LAPACK: /home/travis/R-bin/lib/R/lib/libRlapack.so</span></a>
<a class="sourceLine" id="cb19-9" data-line-number="9"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb19-10" data-line-number="10"><span class="co">#&gt; locale:</span></a>
<a class="sourceLine" id="cb19-11" data-line-number="11"><span class="co">#&gt;  [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C              </span></a>
<a class="sourceLine" id="cb19-12" data-line-number="12"><span class="co">#&gt;  [3] LC_TIME=en_US.UTF-8        LC_COLLATE=en_US.UTF-8    </span></a>
<a class="sourceLine" id="cb19-13" data-line-number="13"><span class="co">#&gt;  [5] LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   </span></a>
<a class="sourceLine" id="cb19-14" data-line-number="14"><span class="co">#&gt;  [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                 </span></a>
<a class="sourceLine" id="cb19-15" data-line-number="15"><span class="co">#&gt;  [9] LC_ADDRESS=C               LC_TELEPHONE=C            </span></a>
<a class="sourceLine" id="cb19-16" data-line-number="16"><span class="co">#&gt; [11] LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       </span></a>
<a class="sourceLine" id="cb19-17" data-line-number="17"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb19-18" data-line-number="18"><span class="co">#&gt; attached base packages:</span></a>
<a class="sourceLine" id="cb19-19" data-line-number="19"><span class="co">#&gt; [1] stats     graphics  grDevices utils     datasets  methods   base     </span></a>
<a class="sourceLine" id="cb19-20" data-line-number="20"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb19-21" data-line-number="21"><span class="co">#&gt; other attached packages:</span></a>
<a class="sourceLine" id="cb19-22" data-line-number="22"><span class="co">#&gt; [1] alevinQC_1.3.1   BiocStyle_2.15.6</span></a>
<a class="sourceLine" id="cb19-23" data-line-number="23"><span class="co">#&gt; </span></a>
<a class="sourceLine" id="cb19-24" data-line-number="24"><span class="co">#&gt; loaded via a namespace (and not attached):</span></a>
<a class="sourceLine" id="cb19-25" data-line-number="25"><span class="co">#&gt;  [1] tximport_1.15.8      tidyselect_1.0.0     xfun_0.12           </span></a>
<a class="sourceLine" id="cb19-26" data-line-number="26"><span class="co">#&gt;  [4] purrr_0.3.3          colorspace_1.4-1     vctrs_0.2.4         </span></a>
<a class="sourceLine" id="cb19-27" data-line-number="27"><span class="co">#&gt;  [7] htmltools_0.4.0      yaml_2.2.1           rlang_0.4.5         </span></a>
<a class="sourceLine" id="cb19-28" data-line-number="28"><span class="co">#&gt; [10] pkgdown_1.4.1        pillar_1.4.3         later_1.0.0         </span></a>
<a class="sourceLine" id="cb19-29" data-line-number="29"><span class="co">#&gt; [13] glue_1.3.2           RColorBrewer_1.1-2   lifecycle_0.2.0     </span></a>
<a class="sourceLine" id="cb19-30" data-line-number="30"><span class="co">#&gt; [16] plyr_1.8.6           stringr_1.4.0        munsell_0.5.0       </span></a>
<a class="sourceLine" id="cb19-31" data-line-number="31"><span class="co">#&gt; [19] gtable_0.3.0         htmlwidgets_1.5.1    memoise_1.1.0       </span></a>
<a class="sourceLine" id="cb19-32" data-line-number="32"><span class="co">#&gt; [22] evaluate_0.14        labeling_0.3         knitr_1.28          </span></a>
<a class="sourceLine" id="cb19-33" data-line-number="33"><span class="co">#&gt; [25] GGally_1.4.0         fastmap_1.0.1        httpuv_1.5.2        </span></a>
<a class="sourceLine" id="cb19-34" data-line-number="34"><span class="co">#&gt; [28] crosstalk_1.1.0.1    highr_0.8            Rcpp_1.0.4          </span></a>
<a class="sourceLine" id="cb19-35" data-line-number="35"><span class="co">#&gt; [31] xtable_1.8-4         scales_1.1.0         backports_1.1.5     </span></a>
<a class="sourceLine" id="cb19-36" data-line-number="36"><span class="co">#&gt; [34] promises_1.1.0       DT_0.12              BiocManager_1.30.10 </span></a>
<a class="sourceLine" id="cb19-37" data-line-number="37"><span class="co">#&gt; [37] desc_1.2.0           mime_0.9             farver_2.0.3        </span></a>
<a class="sourceLine" id="cb19-38" data-line-number="38"><span class="co">#&gt; [40] fs_1.3.1             rjson_0.2.20         ggplot2_3.3.0       </span></a>
<a class="sourceLine" id="cb19-39" data-line-number="39"><span class="co">#&gt; [43] digest_0.6.25        stringi_1.4.6        bookdown_0.18       </span></a>
<a class="sourceLine" id="cb19-40" data-line-number="40"><span class="co">#&gt; [46] dplyr_0.8.5          shiny_1.4.0.2        grid_4.0.0          </span></a>
<a class="sourceLine" id="cb19-41" data-line-number="41"><span class="co">#&gt; [49] rprojroot_1.3-2      cowplot_1.0.0        tools_4.0.0         </span></a>
<a class="sourceLine" id="cb19-42" data-line-number="42"><span class="co">#&gt; [52] magrittr_1.5         tibble_2.1.3         crayon_1.3.4        </span></a>
<a class="sourceLine" id="cb19-43" data-line-number="43"><span class="co">#&gt; [55] pkgconfig_2.0.3      MASS_7.3-51.5        shinydashboard_0.7.1</span></a>
<a class="sourceLine" id="cb19-44" data-line-number="44"><span class="co">#&gt; [58] assertthat_0.2.1     rmarkdown_2.1        reshape_0.8.8       </span></a>
<a class="sourceLine" id="cb19-45" data-line-number="45"><span class="co">#&gt; [61] R6_2.4.1             compiler_4.0.0</span></a></code></pre></div>
</div>
<div id="references" class="section level1 unnumbered">
<h1 class="hasAnchor">
<a href="#references" class="anchor"></a>References</h1>
<div id="refs" class="references">
<div id="ref-Srivastava2018-fz">
<p>Srivastava, Avi, Laraib Malik, Tom Sean Smith, Ian Sudbery, and Rob Patro. 2018. “Alevin Efficiently Estimates Accurate Gene Abundances from dscRNA-seq Data.” <em>bioRxiv Doi:10.1101/335000</em>.</p>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="sidebar">

        <div id="tocnav">
      <h2 class="hasAnchor">
<a href="#tocnav" class="anchor"></a>Contents</h2>
      <ul class="nav nav-pills nav-stacked">
<li><a href="#introduction">Introduction</a></li>
      <li><a href="#installation">Installation</a></li>
      <li><a href="#assumed-output-directory-structure">Assumed output directory structure</a></li>
      <li><a href="#check-that-all-required-alevin-files-are-available">Check that all required alevin files are available</a></li>
      <li><a href="#generate-qc-report">Generate QC report</a></li>
      <li><a href="#create-shiny-app">Create shiny app</a></li>
      <li><a href="#generate-individual-plots">Generate individual plots</a></li>
      <li><a href="#session-info">Session info</a></li>
      <li><a href="#references">References</a></li>
      </ul>
</div>
      </div>

</div>



      <footer><div class="copyright">
  <p>Developed by <a href="https://csoneson.github.io">Charlotte Soneson</a>, Avi Srivastava.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.4.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
