<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>alevinQC • alevinQC</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/cosmo/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><meta property="og:title" content="alevinQC">
<meta property="og:description" content="alevinQC">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">alevinQC</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.7.1</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="../articles/alevinqc.html">alevinQC</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/csoneson/alevinQC/">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><script src="alevinqc_files/accessible-code-block-0.0.1/empty-anchor.js"></script><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>alevinQC</h1>
                        <h4 class="author">Charlotte Soneson</h4>
            
            <h4 class="date">2021-01-31</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/csoneson/alevinQC/blob/master/vignettes/alevinqc.Rmd"><code>vignettes/alevinqc.Rmd</code></a></small>
      <div class="hidden name"><code>alevinqc.Rmd</code></div>

    </div>

    
        <div class="abstract">
      <p class="abstract">Abstract</p>
      <p><em><a href="https://bioconductor.org/packages/3.13/alevinQC">alevinQC</a></em> reads output files from alevin and generates summary reports.</p>
    </div>
    
<div id="introduction" class="section level1">
<h1 class="hasAnchor">
<a href="#introduction" class="anchor"></a>Introduction</h1>
<p>The purpose of the <em><a href="https://bioconductor.org/packages/3.13/alevinQC">alevinQC</a></em> package is to generate a summary QC report based on the output of an <a href="https://salmon.readthedocs.io/en/latest/alevin.html">alevin</a> <span class="citation">(Srivastava et al. 2019)</span> run. The QC report can be generated as a html or pdf file, or launched as a shiny application.</p>
</div>
<div id="installation" class="section level1">
<h1 class="hasAnchor">
<a href="#installation" class="anchor"></a>Installation</h1>
<p><code>alevinQC</code> can be installed using the <code>BiocManager</code> CRAN package.</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw">if</span> <span class="op">(</span><span class="op">!</span><span class="fu"><a href="https://rdrr.io/r/base/ns-load.html">requireNamespace</a></span><span class="op">(</span><span class="st">"BiocManager"</span>, quietly <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span><span class="op">)</span>
    <span class="fu"><a href="https://rdrr.io/r/utils/install.packages.html">install.packages</a></span><span class="op">(</span><span class="st">"BiocManager"</span><span class="op">)</span>
<span class="fu">BiocManager</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/BiocManager/man/install.html">install</a></span><span class="op">(</span><span class="st">"alevinQC"</span><span class="op">)</span></code></pre></div>
<p>After installation, load the package into the R session.</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="kw"><a href="https://rdrr.io/r/base/library.html">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/csoneson/alevinQC">alevinQC</a></span><span class="op">)</span></code></pre></div>
<p>Note that in order to process output from Salmon v0.14 or later, you need Alevin v1.1 or later.</p>
</div>
<div id="assumed-output-directory-structure" class="section level1">
<h1 class="hasAnchor">
<a href="#assumed-output-directory-structure" class="anchor"></a>Assumed output directory structure</h1>
<p>For more information about running alevin, we refer to the <a href="https://salmon.readthedocs.io/en/latest/alevin.html">documentation</a>. When invoked, alevin generates several output files in the specified output directory. <em><a href="https://bioconductor.org/packages/3.13/alevinQC">alevinQC</a></em> assumes that this structure is retained, and will return an error if it isn’t - thus, it is not recommended to move or rename the output files from alevin. <em><a href="https://bioconductor.org/packages/3.13/alevinQC">alevinQC</a></em> assumes that the following files (in the indicated structure) are available in the provided <code>baseDir</code> (note that currently, in order to generate the full set of files, alevin must be invoked with the <code>--dumpFeatures</code> flag).</p>
<p>For alevin versions before 0.14:</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1"></a>baseDir</span>
<span id="cb3-2"><a href="#cb3-2"></a>  <span class="op">|--</span>alevin</span>
<span id="cb3-3"><a href="#cb3-3"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>featureDump.txt</span>
<span id="cb3-4"><a href="#cb3-4"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>filtered_cb_frequency.txt</span>
<span id="cb3-5"><a href="#cb3-5"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>MappedUmi.txt</span>
<span id="cb3-6"><a href="#cb3-6"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>quants_mat_cols.txt</span>
<span id="cb3-7"><a href="#cb3-7"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>quants_mat_rows.txt</span>
<span id="cb3-8"><a href="#cb3-8"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>quants_mat.gz</span>
<span id="cb3-9"><a href="#cb3-9"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>raw_cb_frequency.txt</span>
<span id="cb3-10"><a href="#cb3-10"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>whitelist.txt</span>
<span id="cb3-11"><a href="#cb3-11"></a>  <span class="op">|--</span>aux_info</span>
<span id="cb3-12"><a href="#cb3-12"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>meta_info.json</span>
<span id="cb3-13"><a href="#cb3-13"></a>  <span class="op">|--</span>cmd_info.json</span></code></pre></div>
<p>For alevin version 0.14 and later:</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1"></a>baseDir</span>
<span id="cb4-2"><a href="#cb4-2"></a>  <span class="op">|--</span>alevin</span>
<span id="cb4-3"><a href="#cb4-3"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>featureDump.txt</span>
<span id="cb4-4"><a href="#cb4-4"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>raw_cb_frequency.txt</span>
<span id="cb4-5"><a href="#cb4-5"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span><span class="kw">whitelist.txt</span>  (depending on how alevin was run)</span>
<span id="cb4-6"><a href="#cb4-6"></a>  <span class="op">|--</span>aux_info</span>
<span id="cb4-7"><a href="#cb4-7"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>meta_info.json</span>
<span id="cb4-8"><a href="#cb4-8"></a>  <span class="op">|</span><span class="st">    </span><span class="er">|</span><span class="op">--</span>alevin_meta_info.json</span>
<span id="cb4-9"><a href="#cb4-9"></a>  <span class="op">|--</span>cmd_info.json</span></code></pre></div>
</div>
<div id="check-that-all-required-alevin-files-are-available" class="section level1">
<h1 class="hasAnchor">
<a href="#check-that-all-required-alevin-files-are-available" class="anchor"></a>Check that all required alevin files are available</h1>
<p>The report generation functions (see below) will check that all the required files are available in the provided base directory. However, you can also call the function <code><a href="../reference/checkAlevinInputFiles.html">checkAlevinInputFiles()</a></code> to run the check manually. If one or more files are missing, the function will raise an error indicating the missing file(s).</p>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">baseDir</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html">system.file</a></span><span class="op">(</span><span class="st">"extdata/alevin_example_v0.14"</span>, package <span class="op">=</span> <span class="st">"alevinQC"</span><span class="op">)</span>
<span class="fu"><a href="../reference/checkAlevinInputFiles.html">checkAlevinInputFiles</a></span><span class="op">(</span>baseDir <span class="op">=</span> <span class="va">baseDir</span><span class="op">)</span>
<span class="co">#&gt; [1] "v0.14"</span></code></pre></div>
</div>
<div id="generate-qc-report" class="section level1">
<h1 class="hasAnchor">
<a href="#generate-qc-report" class="anchor"></a>Generate QC report</h1>
<p>The <code><a href="../reference/alevinQCReport.html">alevinQCReport()</a></code> function generates the QC report from the alevin output. Depending on the file extension of the <code>outputFile</code> argument, and the value of <code>outputFormat</code>, the function can generate either an html report or a pdf report.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">outputDir</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/tempfile.html">tempdir</a></span><span class="op">(</span><span class="op">)</span>
<span class="fu"><a href="../reference/alevinQCReport.html">alevinQCReport</a></span><span class="op">(</span>baseDir <span class="op">=</span> <span class="va">baseDir</span>, sampleId <span class="op">=</span> <span class="st">"testSample"</span>, 
               outputFile <span class="op">=</span> <span class="st">"alevinReport.html"</span>, 
               outputFormat <span class="op">=</span> <span class="st">"html_document"</span>,
               outputDir <span class="op">=</span> <span class="va">outputDir</span>, forceOverwrite <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></code></pre></div>
</div>
<div id="create-shiny-app" class="section level1">
<h1 class="hasAnchor">
<a href="#create-shiny-app" class="anchor"></a>Create shiny app</h1>
<p>In addition to static reports, <em><a href="https://bioconductor.org/packages/3.13/alevinQC">alevinQC</a></em> can also generate a shiny application, containing the same summary figures as the pdf and html reports.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">app</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/alevinQCShiny.html">alevinQCShiny</a></span><span class="op">(</span>baseDir <span class="op">=</span> <span class="va">baseDir</span>, sampleId <span class="op">=</span> <span class="st">"testSample"</span><span class="op">)</span></code></pre></div>
<p>Once created, the app can be launched using the <code>runApp()</code> function from the <em><a href="https://CRAN.R-project.org/package=shiny">shiny</a></em> package.</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu">shiny</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/shiny/man/runApp.html">runApp</a></span><span class="op">(</span><span class="va">app</span><span class="op">)</span></code></pre></div>
</div>
<div id="generate-individual-plots" class="section level1">
<h1 class="hasAnchor">
<a href="#generate-individual-plots" class="anchor"></a>Generate individual plots</h1>
<p>The individual plots included in the QC reports can also be independently generated. To do so, we must first read the alevin output into an R object.</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="va">alevin</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/readAlevinQC.html">readAlevinQC</a></span><span class="op">(</span>baseDir <span class="op">=</span> <span class="va">baseDir</span><span class="op">)</span></code></pre></div>
<p>The resulting list contains four entries:</p>
<ul>
<li>
<code>cbTable</code>: a <code>data.frame</code> with various inferred characteristics of the individual cell barcodes.</li>
<li>
<code>summaryTables</code>: a list of <code>data.frame</code>s with summary information about the full data set, the initial set of whitelisted cells and the final set of whitelisted cells, respectively.</li>
<li>
<code>versionTable</code>: a <code>matrix</code> with information about the invokation of alevin.</li>
<li>
<code>type</code>: a <code>character</code> scalar indicating how alevinQC interpreted the alevin output directory.</li>
</ul>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/head.html">head</a></span><span class="op">(</span><span class="va">alevin</span><span class="op">$</span><span class="va">cbTable</span><span class="op">)</span>
<span class="co">#&gt;                 CB originalFreq ranking collapsedFreq nbrMappedUMI</span>
<span class="co">#&gt; 1 GACTGCGAGGGCATGT       121577       1        123419       104128</span>
<span class="co">#&gt; 2 GGTGCGTAGGCTACGA       110467       2        111987        93608</span>
<span class="co">#&gt; 3 ATGAGGGAGTAGTGCG       106446       3        108173        88481</span>
<span class="co">#&gt; 4 ACTGTCCTCATGCTCC       104794       4        106085        81879</span>
<span class="co">#&gt; 5 CGAACATTCTGATACG       104616       5        106072        84395</span>
<span class="co">#&gt; 6 ACTGTCCCATATGGTC        99208       6        100776        81066</span>
<span class="co">#&gt;   totalUMICount mappingRate dedupRate  MeanByMax nbrGenesAboveZero</span>
<span class="co">#&gt; 1         73312    0.843695  0.295943 0.00735194              7512</span>
<span class="co">#&gt; 2         66002    0.835883  0.294911 0.00783094              7522</span>
<span class="co">#&gt; 3         62196    0.817958  0.297069 0.00832595              7081</span>
<span class="co">#&gt; 4         57082    0.771824  0.302849 0.00619664              6956</span>
<span class="co">#&gt; 5         58547    0.795639  0.306274 0.00743685              7347</span>
<span class="co">#&gt; 6         56534    0.804418  0.302618 0.00947029              6841</span>
<span class="co">#&gt;   nbrGenesAboveMean ArborescenceCount inFinalWhiteList inFirstWhiteList</span>
<span class="co">#&gt; 1              1237           1.42034             TRUE             TRUE</span>
<span class="co">#&gt; 2              1238           1.41826             TRUE             TRUE</span>
<span class="co">#&gt; 3              1151           1.42262             TRUE             TRUE</span>
<span class="co">#&gt; 4               957           1.43441             TRUE             TRUE</span>
<span class="co">#&gt; 5              1238           1.44149             TRUE             TRUE</span>
<span class="co">#&gt; 6              1068           1.43393             TRUE             TRUE</span></code></pre></div>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu">knitr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span><span class="op">(</span><span class="va">alevin</span><span class="op">$</span><span class="va">summaryTables</span><span class="op">$</span><span class="va">fullDataset</span><span class="op">)</span>
<span class="co">#&gt; Warning in kable_pipe(x = structure(c("Total number of processed reads", : The</span>
<span class="co">#&gt; table should have a header (column names)</span></code></pre></div>
<table class="table"><tbody>
<tr class="odd">
<td align="left">Total number of processed reads</td>
<td align="left">7197662</td>
</tr>
<tr class="even">
<td align="left">Number of reads with Ns</td>
<td align="left">35362</td>
</tr>
<tr class="odd">
<td align="left">Number of reads with valid cell barcode (no Ns)</td>
<td align="left">7162300</td>
</tr>
<tr class="even">
<td align="left">Number of mapped reads</td>
<td align="left">4869156</td>
</tr>
<tr class="odd">
<td align="left">Percent mapped (of all reads)</td>
<td align="left">67.65%</td>
</tr>
<tr class="even">
<td align="left">Number of noisy CB reads</td>
<td align="left">1003624</td>
</tr>
<tr class="odd">
<td align="left">Number of noisy UMI reads</td>
<td align="left">266</td>
</tr>
<tr class="even">
<td align="left">Total number of observed cell barcodes</td>
<td align="left">188613</td>
</tr>
</tbody></table>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu">knitr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span><span class="op">(</span><span class="va">alevin</span><span class="op">$</span><span class="va">summaryTables</span><span class="op">$</span><span class="va">initialWhitelist</span><span class="op">)</span>
<span class="co">#&gt; Warning in kable_pipe(x = structure(c("Number of barcodes (initial</span>
<span class="co">#&gt; whitelist)", : The table should have a header (column names)</span></code></pre></div>
<table class="table"><tbody>
<tr class="odd">
<td align="left">Number of barcodes (initial whitelist)</td>
<td align="left">100</td>
</tr>
<tr class="even">
<td align="left">Number of barcodes with quantification (initial whitelist)</td>
<td align="left">100</td>
</tr>
<tr class="odd">
<td align="left">Fraction reads in barcodes (initial whitelist)</td>
<td align="left">84.64%</td>
</tr>
<tr class="even">
<td align="left">Mean number of reads per cell (initial whitelist)</td>
<td align="left">60620</td>
</tr>
<tr class="odd">
<td align="left">Median number of reads per cell (initial whitelist)</td>
<td align="left">58132</td>
</tr>
<tr class="even">
<td align="left">Mean number of detected genes per cell (initial whitelist)</td>
<td align="left">5163</td>
</tr>
<tr class="odd">
<td align="left">Median number of detected genes per cell (initial whitelist)</td>
<td align="left">5268</td>
</tr>
<tr class="even">
<td align="left">Mean UMI count per cell (initial whitelist)</td>
<td align="left">33274</td>
</tr>
<tr class="odd">
<td align="left">Median UMI count per cell (initial whitelist)</td>
<td align="left">31353</td>
</tr>
</tbody></table>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu">knitr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span><span class="op">(</span><span class="va">alevin</span><span class="op">$</span><span class="va">summaryTables</span><span class="op">$</span><span class="va">finalWhitelist</span><span class="op">)</span>
<span class="co">#&gt; Warning in kable_pipe(x = structure(c("Number of barcodes (final whitelist)", :</span>
<span class="co">#&gt; The table should have a header (column names)</span></code></pre></div>
<table class="table"><tbody>
<tr class="odd">
<td align="left">Number of barcodes (final whitelist)</td>
<td align="left">95</td>
</tr>
<tr class="even">
<td align="left">Number of barcodes with quantification (final whitelist)</td>
<td align="left">95</td>
</tr>
<tr class="odd">
<td align="left">Fraction reads in barcodes (final whitelist)</td>
<td align="left">82.39%</td>
</tr>
<tr class="even">
<td align="left">Mean number of reads per cell (final whitelist)</td>
<td align="left">62118</td>
</tr>
<tr class="odd">
<td align="left">Median number of reads per cell (final whitelist)</td>
<td align="left">58725</td>
</tr>
<tr class="even">
<td align="left">Mean number of detected genes per cell (final whitelist)</td>
<td align="left">5260</td>
</tr>
<tr class="odd">
<td align="left">Median number of detected genes per cell (final whitelist)</td>
<td align="left">5343</td>
</tr>
<tr class="even">
<td align="left">Mean UMI count per cell (final whitelist)</td>
<td align="left">34091</td>
</tr>
<tr class="odd">
<td align="left">Median UMI count per cell (final whitelist)</td>
<td align="left">32028</td>
</tr>
</tbody></table>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu">knitr</span><span class="fu">::</span><span class="fu"><a href="https://rdrr.io/pkg/knitr/man/kable.html">kable</a></span><span class="op">(</span><span class="va">alevin</span><span class="op">$</span><span class="va">versionTable</span><span class="op">)</span>
<span class="co">#&gt; Warning in kable_pipe(x = structure(c("Start time", "Salmon version", "Index", :</span>
<span class="co">#&gt; The table should have a header (column names)</span></code></pre></div>
<table class="table"><tbody>
<tr class="odd">
<td align="left">Start time</td>
<td align="left">Thu May 30 13:06:55 2019</td>
</tr>
<tr class="even">
<td align="left">Salmon version</td>
<td align="left">0.14.0</td>
</tr>
<tr class="odd">
<td align="left">Index</td>
<td align="left">/mnt/scratch5/avi/alevin/data/mohu/salmon_index</td>
</tr>
<tr class="even">
<td align="left">R1file</td>
<td align="left">/mnt/scratch5/avi/alevin/data/10x/v2/mohu/100/all_bcs.fq</td>
</tr>
<tr class="odd">
<td align="left">R2file</td>
<td align="left">/mnt/scratch5/avi/alevin/data/10x/v2/mohu/100/all_reads.fq</td>
</tr>
<tr class="even">
<td align="left">tgMap</td>
<td align="left">/mnt/scratch5/avi/alevin/data/mohu/gtf/txp2gene.tsv</td>
</tr>
<tr class="odd">
<td align="left">Library type</td>
<td align="left">ISR</td>
</tr>
</tbody></table>
<p>The plots can now be generated using the dedicated plotting functions provided with <em><a href="https://bioconductor.org/packages/3.13/alevinQC">alevinQC</a></em> (see the help file for the respective function for more information).</p>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/plotAlevinKneeRaw.html">plotAlevinKneeRaw</a></span><span class="op">(</span><span class="va">alevin</span><span class="op">$</span><span class="va">cbTable</span><span class="op">)</span></code></pre></div>
<p><img src="alevinqc_files/figure-html/v13-1.png" width="700"></p>
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/plotAlevinBarcodeCollapse.html">plotAlevinBarcodeCollapse</a></span><span class="op">(</span><span class="va">alevin</span><span class="op">$</span><span class="va">cbTable</span><span class="op">)</span></code></pre></div>
<p><img src="alevinqc_files/figure-html/v13-2.png" width="700"></p>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/plotAlevinQuant.html">plotAlevinQuant</a></span><span class="op">(</span><span class="va">alevin</span><span class="op">$</span><span class="va">cbTable</span><span class="op">)</span></code></pre></div>
<p><img src="alevinqc_files/figure-html/v13-3.png" width="700"></p>
<div class="sourceCode" id="cb18"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="../reference/plotAlevinKneeNbrGenes.html">plotAlevinKneeNbrGenes</a></span><span class="op">(</span><span class="va">alevin</span><span class="op">$</span><span class="va">cbTable</span><span class="op">)</span></code></pre></div>
<p><img src="alevinqc_files/figure-html/v13-4.png" width="700"></p>
</div>
<div id="session-info" class="section level1">
<h1 class="hasAnchor">
<a href="#session-info" class="anchor"></a>Session info</h1>
<div class="sourceCode" id="cb19"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu"><a href="https://rdrr.io/r/utils/sessionInfo.html">sessionInfo</a></span><span class="op">(</span><span class="op">)</span>
<span class="co">#&gt; R Under development (unstable) (2021-01-30 r79911)</span>
<span class="co">#&gt; Platform: x86_64-apple-darwin17.0 (64-bit)</span>
<span class="co">#&gt; Running under: macOS Catalina 10.15.7</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; Matrix products: default</span>
<span class="co">#&gt; BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.dylib</span>
<span class="co">#&gt; LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; locale:</span>
<span class="co">#&gt; [1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; attached base packages:</span>
<span class="co">#&gt; [1] stats     graphics  grDevices utils     datasets  methods   base     </span>
<span class="co">#&gt; </span>
<span class="co">#&gt; other attached packages:</span>
<span class="co">#&gt; [1] alevinQC_1.7.1   BiocStyle_2.19.1</span>
<span class="co">#&gt; </span>
<span class="co">#&gt; loaded via a namespace (and not attached):</span>
<span class="co">#&gt;  [1] Rcpp_1.0.6           assertthat_0.2.1     rprojroot_2.0.2     </span>
<span class="co">#&gt;  [4] digest_0.6.27        mime_0.9             R6_2.5.0            </span>
<span class="co">#&gt;  [7] plyr_1.8.6           evaluate_0.14        ggplot2_3.3.3       </span>
<span class="co">#&gt; [10] highr_0.8            pillar_1.4.7         rlang_0.4.10        </span>
<span class="co">#&gt; [13] jquerylib_0.1.3      DT_0.17              rmarkdown_2.6       </span>
<span class="co">#&gt; [16] pkgdown_1.6.1.9000   textshaping_0.2.1    desc_1.2.0          </span>
<span class="co">#&gt; [19] labeling_0.4.2       stringr_1.4.0        htmlwidgets_1.5.3   </span>
<span class="co">#&gt; [22] munsell_0.5.0        shiny_1.6.0          compiler_4.1.0      </span>
<span class="co">#&gt; [25] httpuv_1.5.5         xfun_0.20            pkgconfig_2.0.3     </span>
<span class="co">#&gt; [28] systemfonts_0.3.2    htmltools_0.5.1.1    tidyselect_1.1.0    </span>
<span class="co">#&gt; [31] tximport_1.19.2      tibble_3.0.6         bookdown_0.21       </span>
<span class="co">#&gt; [34] reshape_0.8.8        crayon_1.4.0         dplyr_1.0.3         </span>
<span class="co">#&gt; [37] later_1.1.0.1        grid_4.1.0           jsonlite_1.7.2      </span>
<span class="co">#&gt; [40] xtable_1.8-4         GGally_2.1.0         gtable_0.3.0        </span>
<span class="co">#&gt; [43] lifecycle_0.2.0      magrittr_2.0.1       scales_1.1.1        </span>
<span class="co">#&gt; [46] stringi_1.5.3        cachem_1.0.1         farver_2.0.3        </span>
<span class="co">#&gt; [49] fs_1.5.0             promises_1.1.1       bslib_0.2.4         </span>
<span class="co">#&gt; [52] ellipsis_0.3.1       ragg_0.4.1           generics_0.1.0      </span>
<span class="co">#&gt; [55] vctrs_0.3.6          cowplot_1.1.1        rjson_0.2.20        </span>
<span class="co">#&gt; [58] RColorBrewer_1.1-2   tools_4.1.0          glue_1.4.2          </span>
<span class="co">#&gt; [61] purrr_0.3.4          crosstalk_1.1.1      fastmap_1.1.0       </span>
<span class="co">#&gt; [64] yaml_2.2.1           colorspace_2.0-0     BiocManager_1.30.10 </span>
<span class="co">#&gt; [67] shinydashboard_0.7.1 memoise_2.0.0        knitr_1.31          </span>
<span class="co">#&gt; [70] sass_0.3.1</span></code></pre></div>
</div>
<div id="references" class="section level1 unnumbered">
<h1 class="hasAnchor">
<a href="#references" class="anchor"></a>References</h1>
<div id="refs" class="references">
<div id="ref-Srivastava2019-fz">
<p>Srivastava, Avi, Laraib Malik, Tom Sean Smith, Ian Sudbery, and Rob Patro. 2019. “Alevin Efficiently Estimates Accurate Gene Abundances from dscRNA-seq Data.” <em>Genome Biology</em> 20: 65.</p>
</div>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

        <nav id="toc" data-toggle="toc"><h2 data-toc-skip>Contents</h2>
    </nav>
</div>

</div>



      <footer><div class="copyright">
  <p>Developed by <a href="https://csoneson.github.io">Charlotte Soneson</a>, Avi Srivastava.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.9000.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
